lib/sink.js

```diff
@@ -8,6 +8,7 @@ const isString = require('lodash.isstring');
 const common = require('asset-pipe-common');
 const stream = require('readable-stream');
 const assert = require('assert');
+const extname = require('ext-name');
```

Comment:
what about https://www.npmjs.com/package/mime-types instead?

Reply:
yea that looks nicer

Reply:
done